pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -304,18 +304,18 @@ type APIServerSettings struct {
 	EndpointReconcilingDisabled *bool `json:"endpointReconcilingDisabled,omitempty"`
 }
 
-type DeploymentSettings struct {
+type EtcdSettings struct {
```

Comment:
This is breaking backward compatibility without IMO bringing much value. I would rather avoid it.